version.php

```diff
@@ -29,7 +29,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2020110300;  // Plugin version.
+$plugin->version = 2022122000;  // Plugin version.
```

Comment:
I would lean towards making this a micro bump instead of full date bump.

- It's a modification on a fork of the canonical plugin, so handling of version and dates might differ.
- It keeps the changes closer to the date it originates from and whilst this might change before it is merged in, it allows for any major version upstream changes to be OK for updates from our fork, in case of an upstream fork desync.

e.g. upstream changes to 2022121900. If we update to 2022122000 we might miss out on some upgrades in between those versions. If we play it more safe and use micro bumps, this is less likely to happen. Of course there might be conflict resolution in either case, but makes it less impactful for current clients using our fork.

Ideally, the changes will be merged upstream (after any necessary version tweaks), but they will always be in the future and apply changes as expected.

Reply:
Ah understood, this initially wasn't something I considered, thanks for the insight. Silly question, am I fine to bump the plugin version by just a day?